vogue/commands/load/application_tag.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import click
+import json
+from vogue.load.application_tag import load_aplication_tags
+
+
+LOG_LEVELS = ['DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL']
+LOG = logging.getLogger(__name__)
+
+
+@click.command("apptag", short_help="Reads database json dump with application tag info.")
```

Comment:
Any json will do ;) from a database, dumped, cli, piped, from file, or transmitted by manipulating the flapping of butterfly wings.

```suggestion
@click.command("apptag", short_help="Reads  json with application tag info.")
```